### ISSUE TITLE
cpuTemperature not working

### ISSUE THREAD

**Issue by KentLeong** [sebhildebrandt/systeminformation#375]
**Describe the bug**
si.cpuTemperature() does not properly show temps

**To Reproduce**
`si.cpuTemperature(data => { console.log(data)})`

**Current Output**
If applicable, add output to help explain your problem.
`{ main: -1, cores: [], max: -1}`

**Expected behavior**
show actual temps

**Environment (please complete the following information):**
 - systeminformation package version: ^4.26.1
 - OS: Ubuntu 20.04 LTS
 - Hardware: 
mobo: B450 AORUS M
cpu: AMD Ryzen 7 1800X Eight-Core Processor 3.6GHz

**Additional context**
I tried installing `sudo apt-get install lm-sensors`
when I `sensors`, the output is:
```
k10temp-pci-00c3
Adapter: PCI adapter
Tdie:         +25.0°C  (high = +70.0°C)
Tctl:         +45.0°C  

```

I'm not sure what the issue is because I have 4 other computers that I used the same exact methods and all of them are having no issues except this one.

Edit: Also forgot to mention every computer is using a different mobo/cpu

**Comment by sebhildebrandt** [comment]
@KentLeong Thank you for discovering and reporting! I had a similar problem three weeks ago and I thought it is fixed (unfortunately we see that there is still a problem on AMD boards) 

Can you provide the output of

```
cat /sys/class/hwmon/hwmon1/temp*_la*;echo "---";cat /sys/class/hwmon/hwmon1/temp*_i*'
```

Thank you in advance! The output `sensors` you already posted, so I think, there is a way for a fix ;-)

**Comment by KentLeong** [comment]
@sebhildebrandt Thanks for your speedy response!

It seems that the issue was fixed after restarting the server haha..

I outputted the results for this for your reference if you still wanted to see though:
```
cd  /sys/class/hwmon/hwmon1/
ls -lsa
```

```
0 drwxr-xr-x 3 root root    0 May 21 05:12 .
0 drwxr-xr-x 3 root root    0 May 21 05:12 ..
0 lrwxrwxrwx 1 root root    0 May 21 05:12 device -> ../../../0000:00:18.3
0 -r--r--r-- 1 root root 4096 May 21 05:12 name
0 drwxr-xr-x 2 root root    0 May 21 19:14 power
0 lrwxrwxrwx 1 root root    0 May 21 05:12 subsystem -> ../../../../../class/hwmon
0 -r--r--r-- 1 root root 4096 May 21 05:12 temp1_input
0 -r--r--r-- 1 root root 4096 May 21 05:12 temp1_label
0 -r--r--r-- 1 root root 4096 May 21 05:12 temp1_max
0 -r--r--r-- 1 root root 4096 May 21 05:12 temp2_input
0 -r--r--r-- 1 root root 4096 May 21 05:12 temp2_label
0 -rw-r--r-- 1 root root 4096 May 21 05:12 uevent

```

**Comment by komorebi-san** [comment]
Hi guys, 

I have the same issue. Here is the sensors output

```
amdgpu-pci-0400
Adapter: PCI adapter
vddgfx:           N/A  
vddnb:            N/A  
edge:         +37.0°C  (crit = +80.0°C, hyst =  +0.0°C)

k10temp-pci-00c3
Adapter: PCI adapter
Tdie:         +37.2°C  (high = +70.0°C)
Tctl:         +37.2°C  
```

And also

```
total 0
0 drwxr-xr-x 3 root root    0 May 27 17:13 .
0 drwxr-xr-x 4 root root    0 May 27 17:13 ..
0 lrwxrwxrwx 1 root root    0 May 27 17:13 device -> ../../thermal_zone0
0 -r--r--r-- 1 root root 4096 May 27 17:13 name
0 drwxr-xr-x 2 root root    0 May 27 17:45 power
0 lrwxrwxrwx 1 root root    0 May 27 17:13 subsystem -> ../../../../../class/hwmon
0 -r--r--r-- 1 root root 4096 May 27 17:36 temp1_input
0 -rw-r--r-- 1 root root 4096 May 27 17:13 uevent
```

**Comment by sebhildebrandt** [comment]
@komorebi-san can you post the output of `temp1_input`?

**Comment by sebhildebrandt** [comment]
I just released a new version (4.26.5) which anyway should fix the issue. @komorebi-san can you confirm that the new version shows temperature values on your machine now?

**Comment by komorebi-san** [comment]
@sebhildebrandt Thank you so much! I have updated to the latest version (4.26.5) and it works now.
